### DISCUSSION TITLE
Add Command to reset all rule timers at once.

### DISCUSSION THREAD

**Discussion by cluelesscris** [arendst/Tasmota#10338] [Ideas]
Whilst developing a strategy for remote PIRs to trigger one or more led lights/strips with timed on periods and fade outs I had a need to reset all timers when a new motion came in whilst within fade outs (which are stepped so use multiple timers). 

Rather than a rule backlog having 'ruletimer1 0; ruletimer2 0;ruletimer3 0 ... It would be useful to to have a single command to quickly reset all timers immediately.

**Comment by barbudor** [comment]
Looking into it

**Reply by barbudor** [reply]
@cluelesscris 

Done and merger by Theo 
https://github.com/arendst/Tasmota/pull/10352
Available in next build of the development branch

`RuleTimer0 0` will clear all timers  (`Rule0 time` will set all timers to the same time)

**Reply by cluelesscris** [reply]
Wow!  Thank you.

**Reply by barbudor** [reply]
Can you please mark the discussion as "answered" ? Thanks

**Reply by cluelesscris** [reply]
Sorry.  New to this.  I can't quite see how to do that.
